tests/conftest.py

```diff
@@ -54,7 +54,10 @@ def model_configs(evaluation_config, task_config):
             "spacy/da_core_news_sm",
         ],
         "question-answering": ["deepset/minilm-uncased-squad2"],
-        "offensive-text-classification": ["DaNLP/da-electra-hatespeech-detection"],
+        "offensive-text-classification": [
+            "alexandrainst/da-electra-hatespeech-detection"
```

Comment:
Changed the model name again, sorry. All models in our repo now follow the format "<language>-<task>-<size>".

```suggestion
            "alexandrainst/da-hatespeech-detection-small"
```